src/themes/default/pages/Category.vue

```diff
@@ -93,12 +93,14 @@ import onBottomScroll from '@vue-storefront/core/mixins/onBottomScroll'
 import rootStore from '@vue-storefront/core/store';
 import { catalogHooksExecutors } from '@vue-storefront/core/modules/catalog-next/hooks'
 
+const THEME_PAGE_SIZE = 3
```

Comment:
just 3 elements? are you sure? it should be 50 here as it was previously

Reply:
Forgot about that. Fixed.